src/main/java/hudson/remoting/Util.java

```diff
@@ -247,4 +251,26 @@ static InetSocketAddress getResolvedHttpProxyAddress(String host, int port) thro
         }
         return targetAddress;
     }
+
+    @IgnoreJRERequirement @SuppressWarnings("Since15")
+    static void mkdirs(@Nonnull File file) throws IOException {
+        if (file.isDirectory()) return;
+
+        try {
+            Class.forName("java.nio.file.Files");
+            Files.createDirectories(file.toPath());
+            return;
+        } catch (ClassNotFoundException e) {
+            // JDK6
```

Comment:
Sure. I've forgotten you're on 1.580.
Right now there is no strong need to migrate, so I would not like to hurry too much